README.md

```diff
@@ -1 +1,33 @@
-# vocabulary
\ No newline at end of file
+
+# IDR Controlled Vocabularies and Ontology Annotation
+
+We are using a number of controlled vocabularies and ontologies in the BBSRC-funded [Image Data Repository](http://idr-demo.openmicroscopy.org/webclient/userdata/?experimenter=-1) (IDR) project.  Our approaches to annotation are being modified and expanded as we gather more types of imaging datasets, expand the functionality we are providing within OMERO for the IDR,  and as a result of discussions with others about the best approaches to take.  We have a spreadsheet outlining the [controlled vocabulary and ontology sources](https://docs.google.com/spreadsheets/d/1S9of23dD8vY1QUv90RV_-Ugu0h6yTeNobuj92-OoSl8/edit?usp=sharing) for terms we are currently using.
```

Comment:
The controlled vocabulary and ontology sources spreadsheet looks like it's for "internal use only" and not intended for external info? E.g. In most table cells the content is too long so I have to look at the top of the page to read the text in the currently selected cell. And the layout is quite haphazard etc. I wonder if (in due course) this could be moved into a more readable format, E.g. a markdown file that simply used a hierarchical bullet points for each Tag, with nested terms (linking to their ontologies)?